src/mmw/apps/home/views.py

```diff
@@ -137,8 +138,10 @@ def get_client_settings(request):
             'mapshed_max_area': settings.GWLFE_CONFIG['MaxAoIArea'],
             'data_catalog_enabled': bigcz,
             'itsi_enabled': not bigcz,
+            'title': title,
         }),
         'google_maps_api_key': settings.GOOGLE_MAPS_API_KEY,
+        'title': title,
```

Comment:
We need this outer one for Django templates like `templates/head.html`. The `client_settings` one is used in JavaScript.